src/utilities/findBreakingChanges.js

```diff
@@ -496,6 +498,41 @@ function isChangeSafeForInputObjectFieldOrFieldArg(
   return false;
 }
 
+function isChangeSafeForDefaultValue(
+  oldDefaultValue: ?mixed,
+  newDefaultValue: ?mixed,
+): boolean {
+  if (oldDefaultValue === undefined) {
+    return true;
+  }
+
+  if (oldDefaultValue === newDefaultValue) {
+    return true;
+  }
+
+  if (
+    oldDefaultValue != null &&
+    newDefaultValue != null &&
+    typeof oldDefaultValue === 'object' &&
+    typeof newDefaultValue === 'object'
+  ) {
+    if (Array.isArray(oldDefaultValue) !== Array.isArray(newDefaultValue)) {
+      return false;
+    }
+
+    for (const key in oldDefaultValue) {
+      if (
+        !isChangeSafeForDefaultValue(oldDefaultValue[key], newDefaultValue[key])
+      ) {
+        return false;
+      }
```

Comment:
@mjmahone According to current implementation adding properties to object is safe.
For example `isChangeSafeForDefaultValue({}, {a: 1}) => true` or `isChangeSafeForDefaultValue({a: undefined}, {a: 1}) => true`.

Reply:
I second this. Can we add a couple more testcases to make sure that check works correctly?

Reply:
That should be correct: adding a previously undefined value is OK, but removing a previously undefined value is not (recursively).